RobotBaseCodes/RobotBaseCodes.ino

```diff
@@ -358,17 +358,29 @@ void IR_reading(IR_SENSOR sensor)
   switch (sensor)
   {
     case LEFT_MID: 
-      SerialCom->println("Mid Left IR Sensor");
-      SerialCom->println(MID_RANGE_LEFT_PIN);
+      SerialCom->print("Mid Left IR Sensor: ");
+      SerialCom->print((pow(2112.5, 1/0.972)) / (pow(analogRead(MID_RANGE_LEFT_PIN),1/0.972))); //MID_RANGE_LEFT_PIN
+      SerialCom->println(" Cm");
+      SerialCom->println(analogRead(MID_RANGE_LEFT_PIN));
```

Comment:
do we need to print the un-adjusted analog reading as well?? Seems redundant now as were already printing the more accurate calibrated value on line 362.